templates/dashboard.php

```diff
@@ -1,11 +1,18 @@
 <?php
 $arrHostapdConf = parse_ini_file('/etc/raspap/hostapd.ini');
 if ($arrHostapdConf['WifiAPEnable'] == 1) {
-    $client_iface = 'uap0';
+  $client_iface = 'uap0';
 } else {
-    $client_iface = RASPI_WIFI_CLIENT_INTERFACE;
+  $client_iface = RASPI_WIFI_CLIENT_INTERFACE;
+}
+$MACPattern = '"([[:xdigit:]]{2}:){5}[[:xdigit:]]{2}"';
+if ($arrHostapdConf['BridgedEnable'] == 1) {
+  $moreLink = "index.php?page=hostapd_conf";
+  exec('iw dev '.$client_iface.' station dump | grep -oE '.$MACPattern, $clients);
+} else {
+  $moreLink = "index.php?page=dhcpd_conf";
+  exec('cat '.RASPI_DNSMASQ_LEASES.'| grep -E $(arp -i '.$client_iface.' -n | grep -oE '.$MACPattern.' | paste -sd "|")', $clients);
```

Comment:
```suggestion
  exec('cat '.RASPI_DNSMASQ_LEASES.'| grep -E $(iw dev '.$client_iface.' station dump | grep -oE '.$MACPattern.' | paste -sd "|")', $clients);
```

iw station dump is more up-to-date than arp cache